Library/Homebrew/dev-cmd/pr-automerge.rb

```diff
@@ -40,6 +43,8 @@ def pr_automerge_args
   def pr_automerge
     args = pr_automerge_args.parse
 
+    odeprecated "`brew pr-automerge --autosquash`" if args.autosquash?
```

Comment:
Not a huge deal but I wonder if it would be helpful to show that the replacement is to just run without the arg:

```suggestion
    odeprecated "`brew pr-automerge --autosquash`", "`brew pr-automerge`" if args.autosquash?
```

That also might be too long and complicated and since this a developer command its probably not a big deal to just expect the user to look at the help text

Reply:
Yea, I don't think it hurts to show the replacement. Let's show that here and in `pr-publish`.